build.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0"?>
+
+<project name="PhingGitHub example" default="create-release-assets" basedir="." description="This is an example buildfile for GitHub Phing tasks.">
```

Comment:
This file is in a funny state.

On one hand it is referenced as an example buildfile. It is really not that good an example. Most of it consists of `init` tasks. I would think most projects chose one approach - input or properties file - and left out the other.

On the other hand it is actually fully functional and composed so that it can be used to test the project itself. We do not use it as such though for continuous integration or the like though. Perhaps that could be implemented down the line.